package.json

```diff
@@ -44,12 +44,12 @@
     "node-inspector": "0.12.5"
   },
   "scripts": {
-    "test": "node ./node_modules/mocha/bin/mocha -R spec ./test/*.js",
+    "test": "node ./node_modules/mocha/bin/mocha -S -R spec ./test/*.js",
     "start": "node example/server.js",
-    "coveralls": "node ./node_modules/mocha/bin/mocha --require blanket --reporter mocha-lcov-reporter ./test/*.js | node ./node_modules/coveralls/bin/coveralls.js",
-    "coverage": "node ./node_modules/mocha/bin/mocha --require blanket --reporter html-cov ./test/*.js > coverage.html",
+    "coveralls": "node ./node_modules/mocha/bin/mocha -S --require blanket --reporter mocha-lcov-reporter ./test/*.js | node ./node_modules/coveralls/bin/coveralls.js",
+    "coverage": "node ./node_modules/mocha/bin/mocha -S --require blanket --reporter html-cov ./test/*.js > coverage.html",
     "complexity": "node ./node_modules/plato/bin/plato -r -d complexity lib",
-    "performance": "node --allow-natives-syntax --harmony ./node_modules/mocha/bin/_mocha --reporter mocha-performance ./test/*.js",
+    "performance": "node --allow-natives-syntax --harmony ./node_modules/mocha/bin/_mocha -S --reporter mocha-performance ./test/*.js",
```

Comment:
The `-S` argument tells Mocha to sort the test files and run them in alphabetical order. This ensure the two new tests run before and after all other tests.